Libraries/WebSocket/MessageEvent.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule MessageEvent
+ *
+ */
+
+'use strict';
+
+var Event = require('Event');
```

Comment:
Just curious - this is quite a generic name and could conflict with other code in the future. How about calling it `WebSocketEvent`?